src/sysadmin/installation.md

```diff
@@ -327,16 +328,10 @@ sudo systemctl restart postgresql
 
 ### Java installation { #install_java_installation } 
 
-The recommended Java JDK for DHIS 2 is OpenJDK 11 (for version 2.35 and later). You can install it with the following command:
+The recommended Java JDK for DHIS 2 is OpenJDK 17 (for version 2.41). You can install it with the following command:
```

Comment:
I would actually say "for version 2.40 and later". JDK 17 is well tested and performs a bit better than 11 on 2.40. This way it is also future proof and will be correct in a year or two, we will definitely forget this again in a year :)

Reply:
lovely 👍, i'll update